.github/workflows/wasm-size.yml

```
@@ -0,0 +1,111 @@
+name: Report wasm size
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/**'
+      - '!.github/workflows/query-engine-driver-adapters.yml'
+      - '.buildkite/**'
+      - '*.md'
+      - 'LICENSE'
+      - 'CODEOWNERS'
+      - 'renovate.json'
+
+jobs:
+  pr-wasm-size:
+    name: Get PR module size
+    runs-on: ubuntu-latest
+    outputs:
+      size: ${{ steps.measure.outputs.size }} 
+      size_gz: ${{ steps.measure.outputs.size_gz }} 
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+      
+      - uses: dtolnay/rust-toolchain@stable
+
+      - name: 'Install wasm-pack'
+        run: cargo install wasm-pack
+      
+      - name: Build and measure PR branch
+        id: measure
+        run: |
+          make build-qe-wasm
+          echo "size=$(wc -c < query-engine/query-engine-wasm/pkg/query_engine_bg.wasm)" >> $GITHUB_OUTPUT
+          echo "size_gz=$(gzip -c query-engine/query-engine-wasm/pkg/query_engine_bg.wasm | wc -c)" >> $GITHUB_OUTPUT
+
+  base-wasm-size:
+    name: Get base branch size
+    runs-on: ubuntu-latest
+    outputs:
+      size: ${{ steps.measure.outputs.size }} 
+      size_gz: ${{ steps.measure.outputs.size_gz }} 
+    steps:
+      - name: Checkout base branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+      
+      - uses: dtolnay/rust-toolchain@stable
+
+      - name: 'Install wasm-pack'
+        run: cargo install wasm-pack
+      
+      - name: Build and measure base branch
+        id: measure
+        run: |
+          make build-qe-wasm
+          echo "size=$(wc -c < query-engine/query-engine-wasm/pkg/query_engine_bg.wasm)" >> $GITHUB_OUTPUT
```

```suggestion
          echo "size=$(wc --bytes < query-engine/query-engine-wasm/pkg/query_engine_bg.wasm)" >> $GITHUB_OUTPUT
```
Same as previous comment